package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bellese/angular-design-system",
-  "version": "5.0.0",
+  "version": "5.0.2",
```

Comment:
This is OK, right?

I forget if this is what blows up versions in the GH release bot, or if it's when we change the version manually in `package.json`

Reply:
@mcelestini ? ⬆️ re: when / where to bump version numbers or not?